prime-angular-frontend/src/app/modules/adjudication/shared/components/review-status-content/review-status-content.component.ts

```diff
@@ -123,10 +136,12 @@ export class ReviewStatusContentComponent implements OnInit, OnChanges {
         }
 
         if (esr.statusReasonCode === EnrolmentStatusReasonEnum.IDENTITY_PROVIDER) {
-          return reasons.concat(new Reason(this.configPipe.transform(esr.statusReasonCode, 'statuses'), esr.reasonNote, this.enrollee.identificationDocuments));
+          return reasons.concat(new Reason(this.configPipe.transform(esr.statusReasonCode, 'statusReasons'), esr.reasonNote, this.enrollee.identificationDocuments));
         }
 
-        reasons.push(new Reason(this.configPipe.transform(esr.statusReasonCode, 'statuses'), esr.reasonNote));
+        reasons.push((esr.statusReasonCode === 20)
```

Comment:
I would assume that it would be more appropriate to match using the `EnrolmentStatusReason` enum rather than a magic munber

Reply:
Updated the `EnrolmentStatusReason` enum and remove the magic number